device/cuda/src/seeding/seed_finding.cu

```diff
@@ -201,24 +233,16 @@ vecmem::data::vector_buffer<seed> seed_finding::operator()(
     device::doublet_buffer_pair doublet_buffers = device::make_doublet_buffers(
         doublet_counter_buffer, *m_copy, m_mr.main, m_mr.host);
 
-    // Get the prefix sum for the doublet counter buffer.
-    const device::prefix_sum_t doublet_prefix_sum =
-        device::get_prefix_sum(doublet_counter_buffer.items,
-                               (m_mr.host ? *(m_mr.host) : m_mr.main), *m_copy);
-
-    // Set up the buffer of the prefix sum and its view
-    vecmem::data::vector_buffer<device::prefix_sum_element_t>
-        doublet_prefix_sum_buff(doublet_prefix_sum.size(), m_mr.main);
-    m_copy->setup(doublet_prefix_sum_buff);
-    (*m_copy)(vecmem::get_data(doublet_prefix_sum), doublet_prefix_sum_buff);
-    vecmem::data::vector_view<device::prefix_sum_element_t>
-        doublet_prefix_sum_view = doublet_prefix_sum_buff;
+    // Create prefix sum buffer and its view
+    vecmem::data::vector_buffer doublet_prefix_sum_buff = make_prefix_sum_buff(
+        m_copy->get_sizes(doublet_counter_buffer.items), m_copy, m_mr);
+    vecmem::data::vector_view doublet_prefix_sum_view = doublet_prefix_sum_buff;
```

Comment:
No need for the explicit view here either. Or in any of the next occurrences. But I'll stop pointing these out from here on. :stuck_out_tongue: